src/RoadRegistry.BackOffice.ZipArchiveWriters/ExtractHost/RoadSegmentNationalRoadAttributesToZipArchiveWriter.cs

```diff
@@ -29,9 +30,7 @@ public class RoadSegmentNationalRoadAttributesToZipArchiveWriter : IZipArchiveWr
         if (request == null) throw new ArgumentNullException(nameof(request));
         if (context == null) throw new ArgumentNullException(nameof(context));
 
-        var attributes = await context.RoadSegmentNationalRoadAttributes
-            .InsideContour(request.Contour)
-            .ToListAsync(cancellationToken);
+        var attributes = await context.RoadSegmentNationalRoadAttributes.ToListWithPolygonials(request.Contour, (dbSet, polygon) => dbSet.InsideContour(polygon), x => x.Id, cancellationToken);
```

Comment:
use a line-separated method chain (like what it was)

Reply:
Done